src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,82 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const pages = getNumbers(1, totalPages);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
```

Comment:
Using 'href="#prev"' can lead to unwanted scroll jumps in the user interface. Consider using a button element instead of an anchor for the pagination controls, or prevent the default action of the anchor tag to avoid affecting the page's scroll position.